drivers/cadence/uart/aarch64/cdns_console.S

```diff
@@ -56,29 +56,31 @@ endfunc console_cdns_core_init
 	.globl console_cdns_register
 
 	/* -----------------------------------------------
-	 * int console_cdns_register(uint64_t baseaddr,
```

Comment:
If you are going to change this you have to update the prototype in ``include/drivers/cadence/cdns_uart.h``.